.github/workflows/release.yml

```diff
@@ -1,19 +1,19 @@
-name: Release Package
+name: Release NPM/Docker @latest
 on:
   release:
     types: [ released ]
+
 jobs:
   publish-to-npm:
     runs-on: ubuntu-20.04
-    strategy:
-      matrix:
-        node-version: [14.15.5]
     steps:
       - name: Checkout 🛎️
         uses: actions/checkout@v2
 
-      - uses: actions/setup-node@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v2
         with:
+          node-version: '14.15.5'
```

Comment:
This reminds me need to pick a minimum version of node and update all our repos to use it.  Try running: 
```
find . -type f -name \.tool-versions -exec grep -H nodejs {} \;
```

Reply:
I want to move us to node v16 for everything, but I'm not sure we can yet.